### PR TITLE
Fix bad button position in setup dialog, make the SetupDialog fixed size and EffectDialog resizeable

### LINKED ISSUE #1849: Settings dialog resizable but widgets do not adjust to its new size
The settings dialog is resizable, but the widgets inside it do not adjust to the new size. 

One possibility is to get the widgets to adjust to the new size. Other is to simply make the dialog unresizable.

### PR DESCRIPTION
**Pull request by Umcaruje**
Fixes #1849 

Before & After: 

screenshot from 2016-01-29 16 38 42 screenshot from 2016-01-29 16 38 14

### CODE REVIEW

**Comment by zonkmachine** [comment]
Tested - fixed. There is another thing in there that's a bit odd. If you grab the right edge of the window and expand it to the right it will follow and the OK/Cancel buttons too, but not the text entry fields. The way I see it either they follow or the frame should be fixed.
margin

**Comment by tresf** [comment]
> If you grab the right edge of the window and expand it to the right it will follow and the OK/Cancel buttons too, but not the text entry fields. 

Duplicate of #1849.

**Comment by Umcaruje** [comment]
Ok, so, I decided to make the setup dialog fixed size, so we avoid the resize problems. I also did the same thing for `EffectDialog`. Comments welcome.

**Comment by tresf** [comment]
Looks good although I'm not a fan of all of the code editor auto-formatting changes as they makes the PR harder to read and can really make the cherry-pick process harder down the road.

I'd rather we fix formatting in separate PRs, personally.

**Comment by Umcaruje** [comment]
I'd rather we fix formatting in separate PRs, personally. 

Yeah, I agree, I'll see to disable it in atom and redo this PR.

**Comment by Umcaruje** [comment]
@tresf done. Anything else that I need to change?

**Comment by Umcaruje** [comment]
Ok, so I thought about it, and it really doesn't make sense to have the `RenameDialog` fixed size. So I made it resizeable: 

gifrecord_2016-02-01_210334

As always, testing and comments welcome.

File: src/gui/widgets/RenameDialog.cpp [hunk 1]
```diff
@@ -56,6 +57,13 @@ RenameDialog::~RenameDialog()
 
 
 
+void RenameDialog::resizeEvent (QResizeEvent * event) {
+	m_stringLE->setGeometry ( 10, 5, width() - 20, 20 );
```
Comment (tresf) [inline src/gui/widgets/RenameDialog.cpp]: Side note on this... I feel we shouldn't get too comfortable with the resize hacks, but rather use containers which can handle resizing properly.  That said, looks god.

**Pull request merged**